test/parallel/xyk-pallet.API.payment-info.test.ts

```diff
@@ -114,7 +136,7 @@ test("GIVEN a paymentInfo request, WHEN extrinsic is provideLiquidityWithId THEN
 
 test("GIVEN a paymentInfo request, WHEN extrinsic is a batch with a sell/buy operation THEN non-zero is returned AND the extrinsic will fail because sell/buy are forbidden in batches tx", async () => {
   const batchAllEvent = api.tx.utility.batchAll([
-    Xyk.buyAsset(GASP_ASSET_ID, KSM_ASSET_ID, BN_HUNDRED),
+    Xyk.buyAsset(GASP_ASSET_ID, ETH_ASSET_ID, BN_HUNDRED),
```

Comment:
Xyk.buyAsset -> is now Market.multiswap on Xyk pool type?

Reply:
I have never made Market.buyAsset yet, but I'ĺl change it in any case. I can remove the changes in other tests for now.